### PR TITLE
OSSMDOC-478: distributed tracing 2.2 Release Notes.

### PR DESCRIPTION
**Pull request by JStickler**
Preview here  -> https://deploy-preview-41640--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html

Eng review - rkukura
QE review - jkandasa
Peer review - nalhadef

### CODE REVIEW

File: modules/distr-tracing-document-attributes.adoc [hunk 1]
```diff
@@ -14,15 +14,15 @@
 
 :DTProductName: Red Hat OpenShift distributed tracing
 :DTShortName: distributed tracing
-:DTProductVersion: 2.0
+:DTProductVersion: 2.1
 
 :JaegerName: Red Hat OpenShift distributed tracing platform
 :JaegerShortName: distributed tracing platform
-:JaegerVersion: 1.28.0
+:JaegerVersion: 1.30.1
```
Comment (jkandasa) [inline modules/distr-tracing-document-attributes.adoc:21]: @JStickler JaegerVersion: 1.30.0
```
$ oc get csv
NAME                             DISPLAY                                                 VERSION   REPLACES   PHASE
jaeger-operator.v1.30.0          Red Hat OpenShift distributed tracing platform          1.30.0               Succeeded
opentelemetry-operator.v0.42.0   Red Hat OpenShift distributed tracing data collection   0.42.0               Succeeded
```

File: modules/distr-tracing-rn-new-features.adoc [hunk 2]
```diff
@@ -7,12 +7,27 @@ Feature – Describe the new functionality available to the customer. For enhanc
...
+== New features and enhancements {DTProductName} 2.2.0
+
+This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+=== Component versions supported in {DTProductName} version 2.2.0
+
+|===
+|Component |Version
+|Jaeger
+|1.30.1
```
Comment (jkandasa) [inline modules/distr-tracing-rn-new-features.adoc:25]: @JStickler jaeger: 1.30.0

Comment (rkukura) [inline modules/distr-tracing-rn-new-features.adoc:25]: 1.30.0

**Review by rkukura — CHANGES_REQUESTED** [review]

File: modules/distr-tracing-document-attributes.adoc (hunk 1 above)
Comment (JStickler) [reply modules/distr-tracing-document-attributes.adoc:21]: @jkandasa thanks, I'll update the version.

**Review by jkandasa — APPROVED** [review]
LGTM

**Review by rkukura — APPROVED** [review]
LGTM

**Pull request merged**